packages/redirect/src/class-redirect.php

```diff
@@ -87,6 +89,16 @@ public static function get_url( $source, $args = array() ) {
 			$url = add_query_arg( $to_be_added, $url );
 		}
 
-		return $url;
+		/**
+		 * Filters the return of the Redirect URL.
+		 *
+		 * @since 8.6
```

Comment:
I'd recommend using 8.6.0 to be consistent with other filters

```suggestion
		 * @since 8.6.0
```